vendor/github.com/Juniper/contrail-go-api/keystone.go

```diff
@@ -24,6 +25,11 @@ type KeystoneClient struct {
 	current *KeystoneToken
 }
 
+type KeepaliveKeystoneClient struct {
+	KeystoneClient
+	nextRefresh time.Time
```

Comment:
I'm not sure why we don't use `Issued_At string` - field of `KeystoneToken struct` instead?